### ISSUE TITLE
Fails to allow a Google Drive backup

### ISSUE THREAD

**Issue by arcdev** [koush/support-wiki#197]
Steps:
1. select an app
2. select Backup
3. select Add Cloud Storage Account
4. select Google Drive
=> Immediately presents a dialog stating "Unable to Login. You have not set up any Google Accounts." ![Screenshot_20190414-111217_Helium](https://user-images.githubusercontent.com/7281420/56094996-47946380-5ea6-11e9-9092-39a823ddd0bb.jpg)

I have 4 Google Accounts setup on my device.
![Screenshot_20190414-111338_Settings](https://user-images.githubusercontent.com/7281420/56095034-ac4fbe00-5ea6-11e9-9f76-05c03d61adfc.jpg)

Note: selecting Dropbox presents the Login screen to Dropbox.

**Comment by RainbowMagicMarker** [comment]
Same issue, even unlinked and re-linked the Google Account, no victory.

**Comment by jehali** [comment]
I have the same issue, even unlinked and re-linked the Google Account,

**Comment by hub2git** [comment]
I have the same problem as the OP.

**Comment by KorbinianP** [comment]
Hello, same Issue here. I tried it on my Xiaomi Mi 9 SE (Stock Android 9), Nexus 5 (Android 6, rooted) and Galaxy Tab S3. All the same. Also installing the Premium Key doesn't change the behaviour.

**Comment by TheGoldmasterCorporation** [comment]
this the same problem with android 9 lineage os 16. worked fine with android 5.

**Comment by factor27** [comment]
I found a solution when I was hitting the same issue.

Open helium's permission page and, for some reason, contacts permission was denied.  I enabled the permission and I was able to connect to my google drive account.

**Comment by TheGoldmasterCorporation** [comment]
yes this worked, not sure why, but helium does need access, then just enable access when using privacy guard.
